models/comments.models.js

```diff
@@ -23,9 +23,9 @@ exports.selectCommentsByArticleId = (article_id) => {
             return Promise.reject({status:404 , msg: 'Not Found'})
         }
         const { author, body} = newComment;
-        
+        const inputDate = new Date;
```

Comment:
were you trying to use Date.now() and it wasn't working ?   I don't know what new Date without invocation looks like.